### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by JustJamesTech**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by JustJamesTech — COMMENTED** [review]
All looks good

File: README.md [hunk 1]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
...
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 4 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 4: Suggest changes
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
```
Comment (v-justjamestech) [inline README.md:24]: Could be improved

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (v-justjamestech) [inline index.html:47]: Could be improved

**Review by JustJamesTech — COMMENTED** [review]
Test comment

**Pull request merged**